memgpt/cli/cli_config.py

```diff
@@ -102,6 +103,9 @@ def configure():
     else:
         default_model = "local"  # TODO: figure out if this is ok? this is for local endpoint
 
+    # get the max tokens (context window) for the model
+    default_model_context_window = LLM_MAX_TOKENS[default_model] if default_model in LLM_MAX_TOKENS else LLM_MAX_TOKENS["DEFAULT"]
```

Comment:
don't you want them to specify the token count for the model if its not in the dict?

Reply:
Changed to select from defaults if not openai:

<img width="645" alt="image" src="https://github.com/cpacker/MemGPT/assets/5475622/86b51df3-4499-4358-9647-d0ed8062d29f">